aten/src/ATen/templates/NativeFunctions.h

```diff
@@ -25,6 +25,7 @@
 #include <c10/core/QScheme.h>
 #include <ATen/core/Reduction.h>
 #include <ATen/core/Tensor.h>
+#include <ATen/native/NonSymbolicBC.h>
```

Comment:
The other change to minimize (guarantee?) no internal breakages is here - fbcode uses `NativeFunctions.h` internally, so they should get the headers for the BC symbols automatically.